os_helper.bash

```diff
@@ -139,13 +139,11 @@ tRHSubscribeAttach() {
 tRHEnableEPEL() {
   tIsRHEL || skip "Not required"
   if tIsRHEL 7; then
```

Comment:
So what would be preferred? rpm? I think yum localinstall is needed on EL6 but don't know if it works on EL7.